Example/AztecUITests/Pages/BlogsPage.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+import XCTest
+
+
+class BlogsPage: BasePage {
+    
+    init(appInstance: XCUIApplication) {
+        let expectedElement = appInstance.tables.staticTexts[elementStringIDs.emptyDemo]
+        super.init(appInstance: appInstance, element: expectedElement)
+    }
+    
+    func gotoEmptyDemo() {
+        app.staticTexts[elementStringIDs.emptyDemo].tap()
+        
+        showOptionsStrip()
+    }
+    
+    func gotoDemo() {
+        app.staticTexts[elementStringIDs.demo].tap()
+        
+        showOptionsStrip()
+    }
+    
+    func showOptionsStrip() -> Void {
+        app.textViews[elementStringIDs.richTextField].tap()
+        expandOptionsSctrip()
+    }
+    
+    func expandOptionsSctrip() -> Void {
```

Comment:
On larger devices (e.g. iPad Pro) this function is toggling the extended toolbar open/closed rather than always expanding it. We should check the status of the toolbar before tapping — otherwise the expanded toolbar is toggled off in every other test.

If it isn't currently possible to check the status, we should open an issue to address that as it's also an accessibility issue. (VoiceOver users need to be aware of the toolbar status so they can expand it if needed, too.)

Reply:
#837

Reply:
For now, how about checking to see if the HTML button exists before toggling the extended toolbar? I'm thinking of something like this:

```
        let expandButton = app.children(matching: .window).element(boundBy: 1).children(matching: .other).element.children(matching: .other).element.children(matching: .other).element.children(matching: .button).element
        let htmlButton = app.scrollViews.otherElements.buttons[elementStringIDs.sourcecodeButton]
        
        if expandButton.exists && expandButton.isHittable && !htmlButton.exists {
            expandButton.tap()
        }
```

That way we don't break the existing tests on iPads, and we can improve the check later once that issue is resolved.

Reply:
That's nice solution, thanks!